### PR TITLE
Add nightly run for azure winc 4.16

### PR DESCRIPTION
**Pull request by weinliu**

### CODE REVIEW

**Comment by weinliu** [comment]
@rrasouli could you help take a look?

**Comment by weinliu** [comment]
@liangxia could you help take a look? Thanks

**Comment by weinliu** [comment]
/pj-rehearse pull-ci-openshift-openshift-tests-private-master-debug-winc-azure-ipi

File: ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml [hunk 1]
```diff
@@ -1550,6 +1550,18 @@ tests:
     test:
     - chain: openshift-e2e-test-hypershift-qe
     workflow: cucushift-installer-rehearse-azure-ipi-ovn-hypershift-etcd-disk-encryption-guest
+- as: azure-ipi-ovn-winc-f1
+  cron: 32 0 2,4,6,8,10,12,14,16,18,20,22,24,26,28,30 * *
```
Comment (liangxia) [inline ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml:1554]: Do you really wants to run this so often ? 
BTW, would you mind to make the `fN` and cron entry in sync, currently `fN` is `f1` while cron entry indicates its `f2`.

Comment (weinliu) [reply ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-nightly.yaml]: Updated to use F7, any better now?

**Comment by weinliu** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-azure-ipi-ovn-winc-f7

**Pull request merged**